manifests/plugin/ansible/runner.pp

```diff
@@ -0,0 +1,48 @@
+# = Ansible runner package installation
+#
+# $upstream_runner_repo:: If true, adds upstream repositories to install ansible-runner package from
+#
+class foreman_proxy::plugin::ansible::runner(
+  Boolean $upstream_runner_repo = $foreman_proxy::plugin::ansible::params::upstream_runner_repo,
+) {
+
+  if ( $upstream_runner_repo ) {
+    case $::osfamily {
+      'Debian': {
+        include ::apt
+        ::apt::source { 'ansible-runner':
+          repos    => 'main',
+          location => 'https://releases.ansible.com/ansible-runner/deb',
+          key      => {
+            id     => 'AC48AC71DA695CA15F2D39C4B84E339C442667A9',
+            source => 'https://releases.ansible.com/keys/RPM-GPG-KEY-ansible-release.pub',
+          },
+          include  => {
+            src => false,
+          },
+        }
+        $package = 'ansible-runner'
+      }
+      'RedHat': {
+        yumrepo { 'ansible-runner':
+          descr    => 'Ansible runner',
+          baseurl  => 'https://releases.ansible.com/ansible-runner/rpm/epel-7-x86_64/',
```

Comment:
To make sure it'll also work with EL8 you can use variables. I think there's also one for the architecture
```suggestion
          baseurl  => 'https://releases.ansible.com/ansible-runner/rpm/epel-${facts['os']['release']['major']}-x86_64/',
```

Reply:
well.. not that `ansible-runner` would have builds for others - https://releases.ansible.com/ansible-runner/rpm/ xD but I guess it is for now and they will keep them comming.